earthpy/tests/test_spatial.py

```diff
@@ -42,6 +42,47 @@ def test_extent_to_json():
         es.extent_to_json([0, 1, 1, 0])
 
 
+def test_normalized_diff_shapes():
+    """If provided with arrays of different shapes,
+    normalized_diff will return ValueError"""
+
+    # Test data
+    b1_2d = np.array([[1, 2, 3, 4, 5], [14, 12, 13, 14, 17]])
+    b2_1d = np.array([[6, 7, 8, 9, 10, 16, 17, 18, 19, 20]])
+
+    with pytest.raises(ValueError):
+        normalized_diff(b2=b2_1d, b1=b1_2d)
+
+
+def test_normalized_diff_nomask():
+    """If result of normalized_diff does not include nan or inf values,
+    array is returned as unmasked."""
+
+    # Test data
+    b1_2d = np.array([[1, 2, 3, 4, 5], [14, 12, 13, 14, 17]])
+    b2_2d = np.array([[6, 7, 8, 9, 10], [16, 17, 18, 19, 20]])
+
+    n_diff = normalized_diff(b2=b2_2d, b1=b1_2d)
```

Comment:
This is a great start @jlpalomino looks like it's failing some tests  :)

[see travis results]( https://travis-ci.org/earthlab/earthpy/builds/464008139#L1389 )
To begin you call normalized_diff here. This will work if you do 

 from earthpy.spatial import normalized_diff in the imports. But currently we have all of the spatial module loaded so the better fix for the time being is to just call

`es.normalized_diff()`

Reply:
I also suggest that given we are switching the default band order, that we use that order here. so just swap b1 and b2 parameter / argument order

Reply:
These changes are now reflected in the tests for normalized_diff.